store/presenters/presenters.go

```diff
@@ -53,8 +61,8 @@ func ShowLinkBalance(store *store.Store) (string, error) {
 	}
 	// Because Eth and Link both use 1e18 precision, we can correct using the same facility
 	linkBalance := utils.WeiToEth(balance)
-	result := fmt.Sprintf("Link Balance for %v: %v", address.Hex(), linkBalance)
-	if balance == big.NewInt(0) {
+	result := fmt.Sprintf("Link Balance for %v: %v", address.Hex(), linkBalance.FloatString(18))
+	if balance.Cmp(big.NewInt(0)) == 0 {
```

Comment:
Can we use the `isZero` helper here as well?

Reply:
Do you have a link to help me find the `isZero` helper? I can't find one in our codebase and it doesn't look like `math/big` provides any either.

I can add them if we need to.

Reply:
Added a new one for `BigInt` and pulled them into `utils`